tutorials/quickstarts/README.md

```diff
@@ -0,0 +1,18 @@
+# Quickstarts for Compute Instance
+
+description: Quickstarts for use in Compute Instance
+
+The notebooks in this tutorial will be used as getting started examples on the Notebooks landing page in AML Studio, and will be run in a Compute Instance.
+
+The tutorial consists of three notebooks:
+
+- [AzureMLin10mins.ipynb](AzureMLin10mins.ipynb)
+- [GettingStartedWithPythonSDK.ipynb](GettingStartedWithPythonSDK.ipynb)
```

Comment:
consider ordering alphabetically (this is how they'll be ordered in the main README)

Reply:
Will do.